requirements.txt

```diff
@@ -0,0 +1,5 @@
+ furo
+ myst_parser >=0.13
+ sphinx >=4.0
+ sphinx-copybutton
+ sphinx-autodoc-typehints
```

Comment:
Must: This currently fails to build as we need `sphinx-tags` in the list of requirements